core/model/src/commonMain/kotlin/org/mifos/mobile/core/model/entity/accounts/share/Summary.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2025 Mifos Initiative
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * See https://github.com/openMF/mobile-mobile/blob/master/LICENSE.md
+ */
+package org.mifos.mobile.core.model.entity.accounts.share
+
+import kotlinx.serialization.Serializable
+import org.mifos.mobile.core.model.Parcelable
+import org.mifos.mobile.core.model.Parcelize
+
+@Serializable
+@Parcelize
+data class Summary(
+    val id: Long? = null,
+    val accountNo: String? = null,
+    val productId: Long? = null,
+    val productName: String? = null,
+    val status: Status? = null,
+    val currency: Currency? = null,
+    val timeline: Timeline? = null,
+    val totalApprovedShares: Int? = null,
+    val totalPendingForApprovalShares: Int? = null,
+) : Parcelable
```

Comment:
`@Parcelize` can be removed as shown below. Please remove it from everywhere it is present in this PR.
```suggestion
data class Summary(
    val id: Long? = null,
    val accountNo: String? = null,
    val productId: Long? = null,
    val productName: String? = null,
    val status: Status? = null,
    val currency: Currency? = null,
    val timeline: Timeline? = null,
    val totalApprovedShares: Int? = null,
    val totalPendingForApprovalShares: Int? = null,
)
```